common/src/main/java/com/revenuecat/purchases/common/Backend.kt

```diff
@@ -195,6 +195,7 @@ class Backend(
         val body = mapOf(
             "fetch_token" to purchaseToken,
             "product_ids" to receiptInfo.productIDs,
+            "platform_product_ids" to receiptInfo.platformProductIds?.map { it.toMap },
```

Comment:
could we use the existing `subscriptionOptionId` which is `productId:offerId`?

Reply:
We _could_ but its a little less descriptive when things like RC fortress want to use it 🤷‍♂️